### ISSUE TITLE
failure to uninstall <autogenerated-issue>

### ISSUE THREAD

**Issue by s-aditya-k** [DeterminateSystems/nix-installer#599]
This error occurred after for no visible reason systemd could not find `nix-daemon.service` or `nix-daemon.socket`. I was in the process of reinstaling. running arch linux.

## Error
```
Error: 
   0: Error reverting
   0: Action `create_directory` errored
      Remove file `/nix`

```

## Metadata
|key|value|
|--|--|
|**version**|0.10.0|
|**os**|linux|
|**arch**|x86_64|

**Comment by Hoverbear** [comment]
That's rather interesting! I wonder if the nix store is garbage collecting or something? Hmm.... You said you wer reinstalling, so you ran the uninstall on an existing `nix-installer` installed Nix?

**Comment by s-aditya-k** [comment]
yup! i got it to uninstall fully by rebooting, killing any processes touching `/nix`. i believe that it was probably the daemon that got spawned, despite the fact that the actual `.service` file was missing.
